network_files/config/config.toml

```diff
@@ -209,7 +209,7 @@ laddr = "tcp://0.0.0.0:26656"
 external_address = ""
 
 # Comma separated list of seed nodes to connect to
-seeds = ""
+seeds = "91efb62e89c49e933b530fc15a6b93468a395730@52.204.29.109:26656"
```

Comment:
lets a jq command to start-zetanode.sh to copy this value from 
`network/config/config.toml` -> `~/.zetanode/config/config.toml` ?